spotless.gradle

```diff
@@ -0,0 +1,32 @@
+apply plugin: "com.diffplug.gradle.spotless"
```

Comment:
This is related to the automatic code formatting, right? If this didn't worked out, please remove it.